presentation/src/main/java/com/architect/coders/mu8/utils/PresentationExtensions.kt

```diff
@@ -0,0 +1,52 @@
+package com.architect.coders.mu8.utils
+
+import android.app.Activity
+import android.content.Context
+import android.content.Intent
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import androidx.annotation.LayoutRes
+import androidx.fragment.app.FragmentActivity
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+import androidx.lifecycle.ViewModelProviders
+import com.bumptech.glide.Glide
+
+    fun ImageView.loadUrl(url: String, placeHolder: Int, errorImage: Int) {
```

Comment:
Why this class have added spaces at the beggining?

Reply:
detekt automatic change the class

Reply:
That's not possible, there is no rule or style guide that suggest to do this, please fix this.

Reply:
format the code